wiki/Announcements/Showcases/en.md

```diff
@@ -1,29 +1,27 @@
-Showcases
-=============
+# Showcases
 
-Showcases
-----------
+Showcases on various topics help show the osu!community 
```

Comment:
L3: Please detail since this phrase since it seems to be redundant because of the header.